test_vector_search_in_cloud.py

```diff
@@ -0,0 +1,274 @@
+import ast
+import random
+import time
+from functools import cached_property
+from threading import Thread
+from typing import Optional
+
+import cassandra
+
+from sdcm.mgmt.helpers import get_schema_create_statements_from_file, substitute_dc_name_in_ks_statements_if_different
+from sdcm.tester import ClusterTester
+from sdcm.utils.common import S3Storage
+from sdcm.utils.file import File
+
+# Vector Search test data files
+BUCKET_NAME = "vector-store-in-cloud"
+
+TEST_DATA_FILENAME = "test_data.txt"
+GROUND_TRUTH_FILENAME = "ground_truth.txt"
+DATASET_FILENAME = "dataset.txt"
+SCHEMA_FILENAME = "schema.json"
+
+DATA_DIR_PATH = "data_dir/vector_search/"
+TEST_DATA_FILE_PATH = DATA_DIR_PATH + TEST_DATA_FILENAME
+GROUND_TRUTH_FILE_PATH = DATA_DIR_PATH + GROUND_TRUTH_FILENAME
+DATASET_FILE_PATH = DATA_DIR_PATH + DATASET_FILENAME
+SCHEMA_FILE_PATH = DATA_DIR_PATH + SCHEMA_FILENAME
+
+ANN_SEARCH_CMD_TEMPLATE = "SELECT id FROM vdb_bench.vdb_bench_collection ORDER BY vector ANN OF {vector} LIMIT 10;"
```

Comment:
@mikliapko I would like to see all of the logic and CQL commands needed for validation, backed into latte rune script.

so we can scale it up, and measure pref, and build substantial  cases, for 5min cases  we can spin up dtest environment, of vector store own integration tests can cover that , we can do some basic functional test in SCT, but it's far from being the main goal